src/darwin/Framework/CHIP/MTRDeviceControllerStartupParams.h

```diff
@@ -196,7 +196,9 @@ NS_ASSUME_NONNULL_BEGIN
  *
  * ipk must be 16 bytes in length
  */
-- (instancetype)initWithSigningKeypair:(id<MTRKeypair>)nocSigner fabricId:(uint64_t)fabricId ipk:(NSData *)ipk;
+- (instancetype)initWithSigningKeypair:(id<MTRKeypair>)nocSigner
+                              fabricID:(NSNumber *)fabricID
+                                   ipk:(NSData *)ipk MTR_NEWLY_AVAILABLE;
```

Comment:
If you're fixing the naming to `fabricID` then the last parameter should probably be `IPK:`

Reply:
This is actually going to change to initWithIPK in an upcoming PR that needs to get re-landed, so not going to touch it for now to minimize conflicts.